AdyenComponents/Apple Pay/ApplePayComponentExtensions.swift

```diff
@@ -15,7 +15,7 @@ extension ApplePayComponent: PKPaymentAuthorizationViewControllerDelegate {
     
     /// :nodoc:
     public func paymentAuthorizationViewControllerDidFinish(_ controller: PKPaymentAuthorizationViewController) {
-        if isUserCancel {
```

Comment:
weren't this necessary to differentiate between when user cancels actively and when the payment is finished?

Reply:
It was.. hm..
I'll do one more regression, to check if we break something... :D

Reply:
now `delegate?.didFail(with: ComponentError.cancelled, from: self)` will be called when the payment is complete in addition to when user cancel, right? which merchants were complaining about.

Reply:
I have checked. Seems all good